src/libraries/Common/src/System/Net/CookieParser.cs

```diff
@@ -44,13 +44,13 @@ internal class CookieTokenizer
         private bool _eofCookie;
```

Comment:
I'll hold off on reviewing the PR until the errors and existing questions are addressed.  Additionally, please make sure you update the ref/System.Net.Primitives.cs file... that will be the first thing we look at.

Reply:
Thanks for help! 
ref/System.Net.Primitives.cs is updated.